demo/js/lucca-spe.js

```diff
@@ -10,8 +10,14 @@
 		$scope.authToken;
 		$scope.customFilter = 'hasShortName'; // contains the custom filter selected
 
+		$scope.customInfo = function(user) {
+			var dfd = $q.defer();
+			dfd.resolve(user.id);
+			return dfd.promise;
+		}
+
 		$scope.auth = function(){
-			$http.post("//" + $scope.local + ".local/auth/userlogin?login=passepartout&password=")
+			$http.post("https://" + $scope.local + ".local/auth/userlogin?login=passepartout&password=")
```

Comment:
might create conflicts with master cuz you didn't cherry pick commit from pr 56 https://github.com/LuccaSA/lucca-ui/commit/dab256fab72550d26a3cbd25c3edc15b45e2460a